### PR TITLE
skeleton build and test

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          git checkout 0.9.9.3
+          mkdir build && cd build
+          cmake .. \
+            -GNinja
+          ninja
+          sudo ninja install
+
+      - name: Manually install libsonata
+        run: |
+          git clone https://github.com/BlueBrain/libsonata
```
Comment (matz-e) [inline .github/workflows/run-tests.yml:52]:
```suggestion
          git clone --depth=1 --recursive --shallow-submodules https://github.com/BlueBrain/libsonata
```
to trim things down a wee bit and also fold in the submodule step from below?

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          cd libsonata
+          git submodule update --init --recursive
+          mkdir build && cd build
+          cmake .. \
+            -GNinja \
+            -DEXTLIB_FROM_SUBMODULES=ON \
+            -DSONATA_PYTHON=OFF \
+            -DSONATA_TESTS=OFF
+          ninja
+          sudo ninja install
```
Comment (matz-e) [inline .github/workflows/run-tests.yml:62]:
```suggestion
          cmake \
            -S . \
            -B build \
            -GNinja \
            -DEXTLIB_FROM_SUBMODULES=ON \
            -DSONATA_PYTHON=OFF \
            -DSONATA_TESTS=OFF
          cmake --build build
          sudo cmake --install build
```
As an alternative…

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    runs-on: ubuntu-20.04
+
+    # Run on external PRs, but not internal PRs as they'll be run by the push
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    steps:
+      - name: Install packages
+        run: |
+            sudo apt update
+            sudo apt install                   \
```
Comment (GianlucaFicarelli) [inline .github/workflows/run-tests.yml:16]: Minor, I think that `apt` is suggested for interactive commands, while `apt-get` is still suggested for scripts.
A warning is printed when using `apt`:
```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

Comment (mgeplf) [reply .github/workflows/run-tests.yml]: huh, I did not know that, thanks.

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (mgeplf) [reply .github/workflows/run-tests.yml]: Trying it now...

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (mgeplf) [reply .github/workflows/run-tests.yml:61]: I prefer the way it is now, if you don't mind.

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (mgeplf) [reply .github/workflows/run-tests.yml:52]: it didn't like it, reverting to what it was before.

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (matz-e) [reply .github/workflows/run-tests.yml:62]: Sure, may also require some specific CMake version.

**Review by matz-e — APPROVED** [review]
Looks good, my suggestions (a|we)re optional.

**Comment by mgeplf** [comment]
> Looks good, my suggestions (a|we)re optional.

Always appreciated.

**Pull request merged**